packages/sacado/test/GTestSuite/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ SET(gtest_disable_pthreads ON CACHE BOOL "Disable uses of pthreads in gtest")
 
 # Configure googletest
 add_subdirectory(googletest)
+set_target_properties(sacado-gtest gtest_main PROPERTIES
+  TRIBITS_TESTONLY_LIB TRUE)
```

Comment:
@etphipp, note this small change needed to pull in non-TriBITS raw CMake targets into the updated TriBITS system as part of PR #9894.  We may change this in the future based on feedback and experimentation to determine how to best integration raw CMake packages into TriBITS packages and projects.